libs/stocks/feature-shell/src/lib/stocks/stocks.component.css

```diff
@@ -1,3 +1,7 @@
 mat-form-field {
   display: block;
 }
+
+.stocks-form {
+  padding-bottom: 0.5rem;
```

Comment:
This is required to fix the bottom border issue with the button which was not showing as it was being cut off by the form.
![image](https://user-images.githubusercontent.com/12410864/82754796-10082280-9ded-11ea-8053-68953319fd81.png)
![image](https://user-images.githubusercontent.com/12410864/82754839-52316400-9ded-11ea-9c89-aadeb4971833.png)